lua/starfall/libs_sh/bit.lua

```diff
@@ -499,6 +499,13 @@ function ss_methods:writeString(str)
 	self:write("\0")
 end
 
+--- Writes an entity to the buffer and advances the buffer pointer.
+-- @param Entity e The entity to be written
+function ss_methods:writeEntity(e)
+	self:writeInt16(getent(e):EntIndex())
+	return true
```

Comment:
I'd remove this return

Reply:
Alright. I have it because net.writeEntity() has it.